nms/app/packages/magmalte/app/state/lte/SubscriberState.js

```diff
@@ -238,3 +245,76 @@ export function getSubscriberGatewayMap(subscribers: {[string]: subscriber}) {
   });
   return gatewayMap;
 }
+
+export type SubscriberQueryType = {
+  networkId: string,
+  query: ActionQuery,
+  pageSize: number,
+  tokenList: Array<string>,
+  setTokenList: (Array<string>) => void,
+  ctx: SubscriberContextType,
+  subscriberMetrics?: {[string]: Metrics},
+};
```

Comment:
Please add some field definitions here.

Reply:
Some info that would be helpful to call out:
- What does the `token` in `tokenList` refer to?
- What are `subscriberMetrics` being used for?